readthedocs/projects/models.py

```diff
@@ -251,7 +251,7 @@ class Project(models.Model):
     show_version_warning = models.BooleanField(
         _('Show version warning'),
         default=False,
-        help_text=_('Show warning banner in non-stable nor latest versions.'),
+        help_text=_('Show warning banner in non-stable nor stable latest versions.'),
```

Comment:
This change isn't correct. What we need to add here is the banner links to the stable version, and it's shown only on non-stable versions (except for latest).

Also, you should add new migration (we use named migrations), since it modifies the help_text from the model.

Reply:
I see, I can remove that.